### PR TITLE
Fixes #554 add missing props

### LINKED ISSUE #554: add NumberOfCompilations to reportconfiguraiton

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/OSPSuite.Core/Domain/Constants.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ public static class Constants
    {
       public static readonly int PKML_VERSION = PKMLVersion.Current;
       public const string CVODES = "CVODES";
-      public const int SIM_MODEL_VERSION = 4;
+      public const int SIM_MODEL_VERSION = 5;
```
Comment (msevestre) [inline src/OSPSuite.Core/Domain/Constants.cs]: I have also increased that guy. I didn't know about that field but since we have changed simmodel...

Comment (Yuri05) [reply src/OSPSuite.Core/Domain/Constants.cs]: where is it used?

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Constants.cs]: No idea.. probably export to XML?

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Constants.cs]: https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/blob/f8314455502e3c15ae58510f48d161d593b07651/src/OSPSuite.Core/Serialization/SimModel/DTO/SimulationExport.cs#L25

Comment (Yuri05) [reply src/OSPSuite.Core/Domain/Constants.cs]: OK, so actually it's Simmodel-**XML** version. Should be only increased when XML-structure changes.

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Constants.cs]: ok. I will create a separate issue for that to RENAME the constant and set it to 4

**Pull request merged**